### PR TITLE
Integration refactors

### PR DESCRIPTION
**Pull request by maximiliangonzalez**
merging integration refactors into master

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -69,96 +89,29 @@ Resources:
...
+          DB_PORT: !Ref DatabasePort
+          ENV: !Ref Environment
+          CARS_IMPORT_ERROR_SNS_TOPIC: !Ref CarsImportErrorSnsTopic
           EPSAGON_ACCOUNT_TOKEN: !Ref EpsagonAccountToken
+          EPSAGON_APP_NAME: !Sub volta-${Environment}
       Events:
         Timer:
           Type: Schedule
           Properties:
               Schedule: cron(0 0 1 * ? *)
```
Comment (ryanliszewski) [inline template.yaml:117]: *Non blocking*: Any reservations to increase this cron job to once a day? I'd like the mobile app's car list to always be up to date.

Comment (maximiliangonzalez) [reply template.yaml:117]: No reservations from me. @notbrain @begeekandchic anyone else?

Comment (notbrain) [reply template.yaml:117]: Nope

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+      - *remove_typescript
+      - *save_to_workspace
+
+  deploy:
+    <<: *container_config
+
+    steps:
+      - *install_aws_cli
+      - *configure_aws_cli
+      - *mount_workspace
```
Comment (notbrain) [inline .circleci/config.yml:123]: Nit: imo we should standardize on mounting workspace first.

**Pull request merged**